dts/arm/st/wl/stm32wl.dtsi

```diff
@@ -407,6 +407,8 @@
 			reg = <0x58001000 0x400>;
 			interrupts = <52 0>;
 			clocks = <&rcc STM32_CLOCK_BUS_AHB3 0x00040000>;
+			health-test-magic = <0x17590ABC>;
```

Comment:
when device requires a  magic Nb its value is always 0x17590ABC, 
On the other hand it must not be omitted in the driver even if not present in the DTS.

Reply:
> some the STM32L4 devices also have a RNG Health Test Control and a **LL_RNG_SetHealthconfiguration** function. Should this serie be included here too ?

Yes, some L4 support this health test configuration, but only L4P and L4Q which are not yet supported in zephyr.